pallets/asset-index/src/lib.rs

```diff
@@ -1066,28 +1109,20 @@ pub mod pallet {
 			Ok(())
 		}
 
-		fn remove_saft(
-			who: T::AccountId,
-			asset_id: T::AssetId,
-			units: T::Balance,
-			saft_nav: T::Balance,
-		) -> DispatchResult {
+		fn remove_saft(who: T::AccountId, asset_id: T::AssetId, units: T::Balance, nav: T::Balance) -> DispatchResult {
 			if units.is_zero() {
 				return Ok(());
 			}
 			// native asset can't be processed here
 			Self::ensure_not_native_asset(&asset_id)?;
 
-			// the current index token equivalent value of the given saft nav
-			let index_token = Self::saft_equivalent(saft_nav)?;
-
 			ensure!(!Self::is_liquid_asset(&asset_id), Error::<T>::ExpectedSAFT);
-			ensure!(T::IndexToken::can_slash(&who, index_token), Error::<T>::InsufficientDeposit);
+			ensure!(T::IndexToken::can_slash(&who, nav), Error::<T>::InsufficientDeposit);
```

Comment:
I guess something went wrong when rebasing main?

Reply:
yep...seems my emacs read the file from cache...

Reply:
good catch!